torchbenchmark/util/framework/vision/args.py

```diff
@@ -0,0 +1,37 @@
+import argparse
+import torch
+from torchbenchmark.util.model import BenchmarkModel
+from typing import List, Tuple
+
+def parse_args(model: BenchmarkModel, extra_args: List[str]) -> argparse.Namespace:
+    parser = argparse.ArgumentParser()
+    # by default, enable half precision for inference
+    parser.add_argument("--eval-fp16", action='store_false', help="enable eval fp16")
+    parser.add_argument("--fx2trt", action='store_true', help="enable fx2trt")
+    args = parser.parse_args(extra_args)
+    args.device = model.device
+    args.jit = model.jit
+    args.train_bs = model.train_bs
+    args.eval_bs = model.eval_bs
+    # only enable fp16 in GPU inference
+    if args.device == "cpu":
+        args.eval_fp16 = False
+    return args
+
+def apply_args(model: BenchmarkModel, args: argparse.Namespace):
+    # apply eval_fp16
+    if args.eval_fp16:
+        model.eval_model, model.eval_example_inputs = enable_fp16(model.eval_model, model.eval_example_inputs)
+    # apply fx2trt for eval
+    if args.fx2trt:
+        assert args.device == 'cuda', "fx2trt is only available with CUDA."
+        assert not args.jit, "fx2trt with JIT is not available."
+        model.eval_model = enable_fx2trt(args.eval_bs, args.eval_fp16, model.eval_model, model.eval_example_inputs)
```

Comment:
eval_example_inputs could be a `Tuple[torch.tensor]` (line 31), but you expect example_inputs to be `torch.tensor` (line 34).

Reply:
Fixed type annotation on line 34. Thanks!